### DISCUSSION TITLE
Flowbite Modal

### DISCUSSION THREAD

**Discussion by healer1296** [themesberg/flowbite#378] [Q&A] [answered]
Sorry!!! How can I import Modal? 
<img width="720" alt="Screen Shot 2023-01-02 at 15 05 55" src="https://user-images.githubusercontent.com/66637608/210206625-30d218cf-e41e-4c77-9d5f-2acb0537ce65.png">

**Comment by zoltanszogyenyi** [comment, marked as answer]
Hello @healer1296,

Did you try using:

```
import { Modal } from 'flowbite';
```

Also make sure that you have the latest version of Flowbite installed (v1.6.0).

Cheers,
Zoltan

**Reply by healer1296** [reply]
Thank you very much. They updated docs.
